### ISSUE TITLE
Pod Dashboard isn't functioning correctly

### ISSUE THREAD

**Issue by Krylon360** [jimmidyson/prometheus-grafana-dashboards#3]
Looks like the Template section is pre 3.x. 3.x + uses a new datasource function; and query editor. Datasources; and Dashboards are now Plugin Based.

Could these be rebased to the current Grafana branch?
You can check www.grafana.net for more info.

**Comment by jimmidyson** [comment]
Yeah I'm going to be trying to put together a 3.1 Kubernetes app so this can just be installed via the new plugin interface. All help appreciated :)
